varya/assets/sass/elements/_forms.scss

```diff
@@ -13,12 +13,12 @@ input[type="time"],
 input[type="datetime"],
 input[type="datetime-local"],
 input[type="color"],
+input.wp-block-search__input,
```

Comment:
I think this `input.wp-block-search__input` selector is okay here, but part of me wants this move to the Search Block Sass file itself. Might make it easier to find even if it does mean duplicating the input styles.